src/sync_widgets.rs

```diff
@@ -4,37 +4,28 @@
 
 extern crate gtk;
 
+use gtk::Builder;
 use gtk::prelude::*;
 
 fn main() {
     if gtk::init().is_err() {
         println!("Failed to initialize GTK.");
         return;
     }
+    let glade_src = include_str!("sync_widgets.glade");
+    let builder = Builder::new();
+    builder.add_from_string(glade_src).unwrap();
 
-    let window = gtk::Window::new(gtk::WindowType::Toplevel);
+    let window: gtk::Window = builder.get_object("window").unwrap();
 
-    window.set_title("Enter your age");
-    window.set_position(gtk::WindowPosition::Center);
-    window.set_default_size(300, 20);
-
-    let spin_button = gtk::SpinButton::new_with_range(0.0, 130.0, 1.0);
-    let slider = gtk::Scale::new_with_range(gtk::Orientation::Horizontal, 0.0, 130.0, 1.0);
-
-    let hbox = gtk::Box::new(gtk::Orientation::Horizontal, 5);
-
-    hbox.set_homogeneous(true);
-    hbox.add(&spin_button);
-    hbox.add(&slider);
-
-    window.add(&hbox);
     window.show_all();
```

Comment:
I just moved it down and has no problem. `window` used only in `window.connect_delete_event` so also `get_object` can move there.